client/app/admin/admin.component.ts

```diff
@@ -0,0 +1,154 @@
+import { Component, OnInit } from '@angular/core';
+
+import { AdminService } from './admin.service';
+import { MenuService } from '../menu/menu.service';
+
+@Component({
+    selector: 'app-admin',
+    templateUrl: './admin.component.html',
+    styleUrls: ['./admin.component.css']
+})
+export class AdminComponent implements OnInit {
+
+    public menu = [];
+
+    public staff = [
+        {
+            firstname: 'Klaus',
+            surname: 'Logan',
+            role: 'Waiter'
+        },
+        {
+            firstname: 'Susanne',
+            surname: 'Smith',
+            role: 'Waiter'
+        },
+        {
+            firstname: 'Joanne',
+            surname: 'Lee',
+            role: 'Manager'
+        },
+        {
+            firstname: 'John',
+            surname: 'McGregor',
+            role: 'Manager'
+        },
+        {
+            firstname: 'Lucy',
+            surname: 'McCabe',
+            role: 'Chef'
+        }
+    ];
+
+    private displayOrdersTodayBool;
```

Comment:
These could be:
`private displayOrdersToday: Boolean`

It would keep it consistent with the other components.